src/components/single_use_components/TotalBalance.js

```diff
@@ -1,22 +1,53 @@
-import React from "react";
+import { React, useContext, useState, useEffect } from "react";
 import { View, Text, StyleSheet } from "react-native";
 import { THEME } from "../../constants/Theme";
 import { Ionicons } from "@expo/vector-icons";
 import { TouchableOpacity } from "react-native-gesture-handler";
+import AlgoquantApiContext from "../../constants/ApiContext";
+import LoadSpinner from "../reusable_components/LoadSpinner";
 import { formattedBalance } from "../../helpers/formatUserBalance";
 import { resetBalanceModalBuilder } from "../../helpers/modalFactory";
+import { ConsoleLogger } from "@aws-amplify/core";
```

Comment:
What was this for @WillOrtiz7 ? if not used you can remove

Reply:
> What was this for @WillOrtiz7 ? if not used you can remove

I don't know what that is it can be removed

Reply:
removed